tests/functional/incremental-fetching-empty-rows/expected/data/out/tables/in.c-main.auto-increment-timestamp.csv.gz.manifest

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
Okrem manifestu sa vytvara aj prazdny adresar, vid nizsie `empty-rows/expected/data/out/tables/in.c-main.auto-increment-timestamp.csv.gz/.gitkeep ` ... tj. storage uz tabulku ulozi.

Reply:
Skusal som to aj v KBC, vid screenshoty v PR popise.